src/components/Select/Select.test.tsx

```diff
@@ -23,13 +23,12 @@ describe('<Select />', () => {
 	});
 
 	it('Should set the correct className', () => {
-		const selectComponent = shallow(<Select className={customClass} options={options} />);
+		const selectComponent = mount(<Select className={customClass} options={options} />);
 
-		const selectWrapper = selectComponent.at(0);
-		const selectElement = selectComponent.find('select');
+		const selectElement = selectComponent.find('div').at(0);
 
-		expect(selectWrapper.hasClass(customClass)).toEqual(true);
-		expect(selectWrapper.hasClass('c-select-holder')).toEqual(true);
+		console.log('select element: ', selectElement.html());
```

Comment:
A `console.log` was found wandering the perimeter.

Reply:
https://media1.tenor.com/images/ac6a30be31d9d31b81efb5928716d720/tenor.gif